packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -626,7 +645,11 @@ export async function getExperimentsForActivityFeed(
     }
   );
 
-  return experiments.map((exp) => ({
+  const filteredExperiments = experiments.filter((exp) =>
+    hasReadAccess(context.readAccessFilter, exp.project)
```

Comment:
Same.  This won't work because the above db query does not return a `project` column